app/views/issues/merging.html.erb

```diff
@@ -0,0 +1,56 @@
+<% content_for :title, 'Merge issues' %>
+
+<% content_for :sidebar do %>
+  <%= render 'sidebar'%>
+<% end %>
+
+<div id="issues_editor">
+  <div class="inner note-text-inner">
+    <h3>Merging <%= @issues.count %> issues</h3>
+
+    <%= simple_form_for Issue.new, url: merge_issues_path, method: 'post' do |f| %>
+      <p class="text-warning">WARNING: Only the target issue will prevail, the others will be deleted.</mark>
+      <p>All issues evidences will be moved to the selected issue.
+      <div class="row-fluid">
+        <div class="span6">
+          <p class="merge_option">Select existing issue to merge the others into.
+          <p class="merge_option" style="display: none">Merge issues into a new issue.
+        </div>
+        <div id="change_merge_option" class="span6 pull-right">
+          <a class="merge_option" href="#">Or click here to merge into a new issue</a>
+          <a class="merge_option" style="display: none" href="#">Or click here to merge into a existing issue</a>
+        </div>
+      </div>
+
+      <div class="merge_option">
+        <% checked = true %>
```

Comment:
This `checked` variable is just to make the first radio checked, right? I don't generally like assigning local variables in views if they can be avoided, I think this can be accomplished more succinctly and with clearer intent using `each_with_index`:

```ruby
issues.each_with_index do |issue, i|
  radio_button_tag 'id', issue.id, i == 0
end
```